openedx/core/djangoapps/credit/migrations/0008_auto__del_index_creditprovider_provider_id__chg_field_crediteligibilit.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Changing field 'CreditEligibility.provider'
+        db.alter_column('credit_crediteligibility', 'provider_id', self.gf('django.db.models.fields.related.ForeignKey')(null=True, to=orm['credit.CreditProvider']))
+
+    def backwards(self, orm):
+        # Changing field 'CreditEligibility.provider'
+        db.alter_column('credit_crediteligibility', 'provider_id', self.gf('django.db.models.fields.related.ForeignKey')(default=1, to=orm['credit.CreditProvider']))
```

Comment:
What will setting this to one mean?  If for some reason we have to back out this migration,  what will be the experience of users once the provider_id is set to 1?  Will it seem like everyone needs to verified by whoever is in row one of the providers table?  I might suggest instead that we should seperate this into 2 PRs.

PR 1. Rename the column so that no one uses it. So that a backwards migration can just be to name it back.
PR 2. Delete the column.